[2025-03-11T14:01:49.557+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: project2.import_csv manual__2025-03-11T14:01:48.498328+00:00 [queued]>
[2025-03-11T14:01:49.566+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: project2.import_csv manual__2025-03-11T14:01:48.498328+00:00 [queued]>
[2025-03-11T14:01:49.566+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2025-03-11T14:01:49.567+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2025-03-11T14:01:49.567+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2025-03-11T14:01:49.579+0000] {taskinstance.py:1300} INFO - Executing <Task(PostgresOperator): import_csv> on 2025-03-11 14:01:48.498328+00:00
[2025-03-11T14:01:49.584+0000] {standard_task_runner.py:55} INFO - Started process 572 to run task
[2025-03-11T14:01:49.587+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'project2', 'import_csv', 'manual__2025-03-11T14:01:48.498328+00:00', '--job-id', '229', '--raw', '--subdir', 'DAGS_FOLDER/project2.py', '--cfg-path', '/tmp/tmp_h4t9waq']
[2025-03-11T14:01:49.587+0000] {standard_task_runner.py:83} INFO - Job 229: Subtask import_csv
[2025-03-11T14:01:49.652+0000] {task_command.py:388} INFO - Running <TaskInstance: project2.import_csv manual__2025-03-11T14:01:48.498328+00:00 [running]> on host 2ec744569db4
[2025-03-11T14:01:49.711+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=project2
AIRFLOW_CTX_TASK_ID=import_csv
AIRFLOW_CTX_EXECUTION_DATE=2025-03-11T14:01:48.498328+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-03-11T14:01:48.498328+00:00
[2025-03-11T14:01:49.712+0000] {sql.py:254} INFO - Executing: 
        COPY staging_hr FROM '/tmp/hr_data1.csv'
        DELIMITER ',' CSV HEADER;
        
[2025-03-11T14:01:49.720+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2025-03-11T14:01:49.915+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2025-03-11T14:02:29.985+0000] {local_task_job.py:273} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2025-03-11T14:02:29.987+0000] {process_utils.py:133} INFO - Sending Signals.SIGTERM to group 572. PIDs of all processes in the group: [572]
[2025-03-11T14:02:29.988+0000] {process_utils.py:84} INFO - Sending the signal Signals.SIGTERM to group 572
[2025-03-11T14:03:30.006+0000] {process_utils.py:147} WARNING - process psutil.Process(pid=572, name='airflow task ru', status='sleeping', started='14:01:49') did not respond to SIGTERM. Trying SIGKILL
[2025-03-11T14:03:30.012+0000] {process_utils.py:84} INFO - Sending the signal Signals.SIGKILL to group 572
[2025-03-11T14:03:30.026+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=572, name='airflow task ru', status='terminated', exitcode=<Negsignal.SIGKILL: -9>, started='14:01:49') (572) terminated with exit code Negsignal.SIGKILL
[2025-03-11T14:03:30.027+0000] {standard_task_runner.py:170} ERROR - Job 229 was killed before it finished (likely due to running out of memory)
